### PR TITLE
#162414230 Implement Reset Password Page

### PR DESCRIPTION
**Pull request by tejiri4**
#### What does this PR do?
This PR allows a user to be able to reset a password
#### Description of Task to be completed?
- Create pages for entering an email and resetting existing password
- Connect backend to frontend 

#### How should this be manually tested?
- Git clone the repo
- Cd into it
- run the command npm run dev

#### What are the relevant pivotal tracker stories?

[Reset Password](https://www.pivotaltracker.com/story/show/162414230)

### CODE REVIEW

**Comment by Makwe-O** [comment]
Nothing is working

File: src/reducer/forgot password/forgotPassword.test.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import forgotPasswordReducer from './forgotPassword.reducer';
```
Comment (profmcdan) [inline src/reducer/forgot password/forgotPassword.test.js]: I believe you can optimize the use of variable names while you still ensure that the names remain clear but not necessarily long.

**Review by profmcdan — CHANGES_REQUESTED** [review]
Kindly implement the feedback about naming conventions.

File: src/reducer/forgot password/forgotPassword.test.js (hunk 1 above)
Comment (Makwe-O) [reply src/reducer/forgot password/forgotPassword.test.js]: Thanks for the feedback. I believe the naming still looks good

**Comment by rafmme** [comment]
LGTM, yet

**Review by seunkoko — APPROVED** [review]

**Pull request merged**